cpp/ps0000/problem 0.cpp

```diff
@@ -1,14 +1,15 @@
-#include <iostream>
+#include <iostream> // 2/2
 // first mistake : @ ,, # not #
 // second mistake : "iostream.h" ,, should be iostream without .h and ""
 using namespace std;
 //forgot this line
-int main()
-// he write void and we don't need it ,, Main instead main

```

Comment:
for `void` it's typeless keyword for function declaration and it's used when function don't take any parameters. Here we use `main` function with type `int` but without any parameters